src/modules/enums/Badges.ts

```diff
@@ -110,18 +110,20 @@ enum BadgeEnums {
     'Galar_Water',
     'Galar_Fire',
     'Galar_Fighting',
+    'Galar_Ghost',
     'Galar_Fairy',
     'Galar_Rock',
+    'Galar_Ice',
```

Comment:
I'm not sure how the extra badges will work, or if this is the best way to add them yet.

Reply:
The badge case changes just made allow a region to have 10 badges without it looking weird. If we still want them to be a multiple of 4, we can always merge the badges later, Someone already made images for that.

Reply:
Sorry, wasn't meaning about having 10 badges, just having 2 different gym leaders from 2 different games.

Reply:
It is a little weird, but it's better than just randomly choosing one game, when everything else in pokeclicker either uses expanded versions or mixes both versions. And there's no expanded version for gen 8